Dockerfile

```diff
@@ -4,7 +4,7 @@ ADD ./server/ ./
 ENV PNPM_HOME="/pnpm"
 ENV PATH="$PNPM_HOME:$PATH"
 RUN corepack enable
-RUN pnpm install --frozen-lockfile
+RUN pnpm install --no-frozen-lockfile
```

Comment:
@zoontek `--no-frozen-lockfile` is not a pnpm flag, isn't it ? https://pnpm.io/cli/install

Reply:
@ThomasKientz It is, just not in their documentation. Try `pnpm install --help`:

![Screenshot 2024-11-05 at 10 11 32](https://github.com/user-attachments/assets/25c7ff6b-8b93-4518-9907-506962dde2cb)

Reply:
Thanks! I don't understand why you need it, though. In my understanding, it's better to use `--frozen-lockfile` in CI as you don't want to generate a new lockfile and use the one committed to not deploy new versions of dependencies. 

For the record, we are deploying our own copy of swan-frontend, and we have customized this file as well. Just trying to sync upstream with our repo.

Reply:
Because the clients JS bundle are built and moved to the `server` directory and that's this directory dependencies that are installed and packaged in the Docker image (`ADD ./server/ ./`).

The current pnpm lock is the one for the monorepo, they are none for the server project only (no `pnpm-lock.yaml` in `/server`).

If we set `--frozen-lockfile`, it fails, are no `pnpm-lock.yaml` is present in the copied files. So we explicitely use `--no-frozen-lockfile` exceptionally (acceptable, as we also are also using `save-exact=true`)